HW_03/task2.py

```diff
@@ -0,0 +1,2 @@
+age_in_month =(2022-1977)*12+4
```

Comment:
nit: use spaces for better readability
```suggestion
age_in_month =(2022 - 1977) * 12 + 4
```